.github/workflows/build.yml

```diff
@@ -0,0 +1,36 @@
+on: push
+name: Build
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: ['2', '3']
```

Comment:
You also removed testing on JRuby, which is important to this project.

Reply:
I added `jruby` and `jruby-head` but the build is failing for these due to an error when running `bundle install`

Reply:
This is a known issue. The last version of  the `http_parser.rb` gem that was released for JRuby was 0.6.0. There are known issues in later versions, the most recent work is appearing [on this PR](https://github.com/tmm1/http_parser.rb/pull/75). The fix is to lock `http_parser.rb` to `~> 0.6.0`

Reply:
Related: after the bundle install works, you very likely will see failing test runs under JRuby because of [another bug](https://github.com/TwP/logging/issues/235) I just documented against the Logging gem. The workaround for that is to pin `logging` gem to `~> 2.2.2`

Reply:
I've added these constraints under a separate PR here: https://github.com/adhearsion/adhearsion/pull/656/files.  When that merges we can include it on this branch.